google/ggrc-core
# GGRC-6168 Remove unused os_state column from all tables

# Issue description

Remove unused column os_state from all tables

# Steps to test the changes

1. Run the new migration file '417f46050d33'.
2. Login into mysql server.
3. Reset database using 'db_reset'
4. Verify the presence of 'os_state' column in all tables, with below sql statement.

- select table_name, column_name from information_schema.columns where column_name='os_state';

5. The above statement should return an empty set.

# Solution description

1. Created a new migration file '417f46050d33'
2. Edited the new migration file with required MySQL statements, written under 'upgrade' function.


# Sanity checklist




# PR Review checklist
